packages/lit-dev-content/site/css/tutorial.css

```diff
@@ -107,6 +132,7 @@ litdev-tutorial {
   /* TODO(e111077): I don't know why this nav will not encapsulate the width
   of the buttons inside of it otherwise */
   width: 96px;
```

Comment:
Regarding this TODO - There is a `margin-right: -20px` on `#tutorialHeader #nextButton`. Removing that margin-right allows the removal of this width - I think.

Reply:
Hard to find a good resource. But found some whispers that negative margins behave differently with and without a width specified.

Reply:
PHENOMENAL find!